### PR TITLE
Feature: Mutation (#58433)

### PR DESCRIPTION
**Pull request by nx-mukesh**
#Ticket

https://redmine.successive.tech/issues/58433

### CODE REVIEW

File: apollo-training/src/modules/index.js [hunk 1]
```diff
@@ -2,16 +2,25 @@ import { fileLoader, mergeTypes } from 'merge-graphql-schemas';
...
+import * as trainee from './trainee';
 
 const typesArray = fileLoader(path.join(__dirname, './**/*.graphql'));
-
+console.log('typesArray ---->', typesArray);
+// console.log('typesArray Query', typesArray.Query);
+console.log('user', user);
 const typeDefs = mergeTypes(typesArray, { all: true });
+console.log('typeDefs ---->', typeDefs);
+console.log('user.getMyProfile ---->', user);
```
Comment (permil-successive) [inline apollo-training/src/modules/index.js]: remove consoles

File: apollo-training/src/modules/schema.graphql [hunk 2]
```diff
@@ -1,3 +1,11 @@
 type Query {
   getMyProfile: User!
+  getAllTrainees: [User]
+  getTrainee(id: ID!): User!
```
Comment (permil-successive) [inline apollo-training/src/modules/schema.graphql:4]: Why (!) is used and What is ID type

File: apollo-training/src/server.js [hunk 3]
```diff
@@ -24,11 +24,11 @@ export default class Server {
...
-    this._initHelmet();
-    this._initCompress();
-    this._initCors();
-    this._initJsonParser();
-    this._initMethodOverride();
+    this.initHelmet();
+    this.initCompress();
+    this.initCors();
+    this.initJsonParser();
+    this.initMethodOverride();
```
Comment (permil-successive) [inline apollo-training/src/server.js:32]: Why are you using them?

File: apollo-training/src/service/user.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    return this.users;
+  }
+
+  getUser(id) {
+    return this.users.get(Number(id));
+  }
+
+  createUsers(user) {
+    console.log('user ----> ', user);
+    this.id += 1;
```
Comment (permil-successive) [inline apollo-training/src/service/user.js]: remove console

**Comment by permil-successive** [comment]
apollo-training/package-lock.json dont commit this file.

**Comment by permil-successive** [comment]
Add ticket link as well.

File: apollo-training/src/modules/schema.graphql (hunk 2 above)
Comment (nx-mukesh) [reply apollo-training/src/modules/schema.graphql:4]: (!) exclamation mark is just use for important and required.

File: apollo-training/src/modules/index.js (hunk 1 above)
Comment (nx-mukesh) [reply apollo-training/src/modules/index.js]: done

File: apollo-training/src/service/user.js (hunk 4 above)
Comment (nx-mukesh) [reply apollo-training/src/service/user.js]: Done

**Comment by nx-mukesh** [comment]
Updated as per feedback

File: apollo-training/src/server.js (hunk 3 above)
Comment (nx-mukesh) [reply apollo-training/src/server.js:32]: we are using and initialized some old packages, that'a why we are using this.

File: apollo-training/src/modules/schema.graphql (hunk 2 above)
Comment (permil-successive) [reply apollo-training/src/modules/schema.graphql:4]: ID type ?

**Review by permil-successive — APPROVED** [review]
Approved

**Pull request merged**